build.sbt

```diff
@@ -3878,7 +3878,6 @@ lazy val `engine-runner` = project
             ),
             additionalOptions = Seq(
               "-Dorg.apache.commons.logging.Log=org.apache.commons.logging.impl.NoOpLog",
-              "-H:IncludeResources=.*Main.enso$",
```

Comment:
Doesn't this break `enso --new foo`?

Reply:
Minor: It may be better define this resource in a JSON config file to have all resources in one place.

Reply:
Of course, that's what I was going to do